### ISSUE TITLE
Chrome error pages override

### ISSUE THREAD

**Issue by AndryBray** [nwjs/nw.js#4904]
@rogerwang do you plan any way to override the default chrome error pages/messages (like this one: http://code2care.org/pages/resolving-dns_probe_finished_nxdomain-google-chrome-error/images/DNS_PROBE_FINISHED_NXDOMAIN%20Error%20Message.png) with custom app error pages.

It could be a good feature to let the app look like a native app (and not a well known browser)

Thank you

**Comment by rogerwang** [comment]
Yeah. I'll try to support it. Thanks.

**Comment by iteufel** [comment]
@AndryBray you can try chrome.webRequest.onErrorOccurred

    chrome.webRequest.onErrorOccurred.addListener(function (details) {
		document.getElementsByTagName("iframe")[0].src = "my_error_page"
    },  {urls: ["<all_urls>"]})

**Comment by rogerwang** [comment]
Thanks. @iteufel 's answer should be the solution.

**Comment by AndryBray** [comment]
@iteufel @rogerwang tested this solution with nwjs 0.17.6 and it's not working. there is no iframe inside the default google error page. 
Anyway I'm trying to change the body DOM without success. Is there any other way to do it?
(tested even with some timeout before to set the innerHTML override)

**Comment by AndryBray** [comment]
@rogerwang  @iteufel 
It doesn't work with nw.js 0.17.6 (there is no iframe) and even trying to replace DOM inside the event callback there is no possibility to do it.

I even tested this solution : http://stackoverflow.com/questions/14841256/change-chrome-4xx-page 
but on redirect I get chrome-extension:// invalid error so I get a loop because of the redirect error.

Any other way to accomplish it?
Thank you

**Comment by djcanard** [comment]
I'm also trying to replace the default Chrome error pages with something custom, but I can't get it to work.
I tried chrome.webRequest/webNavigation.onErrorOccurred but these event are not called.
I think maybe that's because I have a Chrome Apps webview in which the page is loaded.

So I tried the loadabort event of the webview. This one is actually triggering but only after the Chrome error page is shown and when the user click on the back button which is on the Chrome error page. So this is too late.

Any ideas?
